platforms/pf-robot-motion-control/include/platform.hpp

```diff
@@ -22,34 +22,33 @@
 #pragma once
 
 /* Project includes */
-#include "canpb/CanProtobuf.hpp"
-#include "utils.hpp"
+#include "pf_common/platform_common.hpp"
+#include "pf_common/uuids.hpp"
 
 /**
  * @name Messages Id
  * @{
+ * @note UUID definitions are centralized in platforms/pf-common/include/pf_common/uuids.hpp
+ *       Add new UUIDs there to ensure consistency across all platforms.
  */
+// Import common UUIDs into global namespace for compatibility
 // Motion Control: 0x1000 - 0x1FFF
-constexpr cogip::canpb::uuid_t state_uuid = 0x1001;
-constexpr cogip::canpb::uuid_t pose_order_uuid = 0x1002;
-constexpr cogip::canpb::uuid_t pose_reached_uuid = 0x1003;
-constexpr cogip::canpb::uuid_t pose_start_uuid = 0x1004;
-constexpr cogip::canpb::uuid_t pid_request_uuid = 0x1005;
-constexpr cogip::canpb::uuid_t pid_uuid = 0x1006;
-constexpr cogip::canpb::uuid_t brake_uuid = 0x1007;
-constexpr cogip::canpb::uuid_t controller_uuid = 0x1008;
-constexpr cogip::canpb::uuid_t blocked_uuid = 0x1009;
-constexpr cogip::canpb::uuid_t intermediate_pose_reached_uuid = 0x100A;
-// Actuators: 0x2000 - 0x2FFF
-// Board: 0xF000 - 0xFFFF
-// Game: 0x4000 - 0x4FFF
-constexpr cogip::canpb::uuid_t game_start_uuid = 0x4001;
-constexpr cogip::canpb::uuid_t game_end_uuid = 0x4002;
-constexpr cogip::canpb::uuid_t game_reset_uuid = 0x4003;
+using cogip::pf_common::blocked_uuid;
+using cogip::pf_common::brake_uuid;
+using cogip::pf_common::controller_uuid;
+using cogip::pf_common::intermediate_pose_reached_uuid;
+using cogip::pf_common::pid_request_uuid;
+using cogip::pf_common::pid_uuid;
+using cogip::pf_common::pose_order_uuid;
+using cogip::pf_common::pose_reached_uuid;
+using cogip::pf_common::pose_start_uuid;
+using cogip::pf_common::state_uuid;
```

Comment:
really useful here ? Should it be inside `pf_common/uuids.hpp` before namespace declaration ?

Reply:
The goal is to explicitly "include" these UUIDs as part of the motion control platform so that they can be used directly within it, like this: `game_reset_uuid`; instead of using `cogip::pf_common::game_reset_uuid;`

This is the way I choose to explicitly declare that certain UUIDs from `pf_common/uuids.hpp` are part of and can be used within this platform.

Reply:
So some kind of "I allow (somehow) this code to use these UUIDs"